src/components/note/NoteEditor.tsx

```
@@ -0,0 +1,144 @@
+  //   // console.log('cleanedContent', );
+  //   console.log('content', value);
+  // };
+
+  const handleChange = (value) => setValue(value);
+  const modules = useMemo(() => {
+    return {
+      toolbar: {
+        container: toolbarOptions,
+      },
+    };
+  }, []);
```

ESLint

[error] React Hook useMemo has a missing dependency: 'toolbarOptions'. Either include it or remove the dependency array.

(react-hooks/exhaustive-deps)